images/windows/Windows2019-Readme.md

```diff
@@ -112,17 +111,17 @@
 
 #### Packages
 - bindgen 0.69.4
-- cargo-audit 0.19.0
+- cargo-audit 0.20.0
 - cargo-outdated 0.14.0
 - cbindgen 0.26.0
 - Clippy 0.1.76
 - Rustfmt 1.7.0
 
 ### Browsers and Drivers
-- Google Chrome 121.0.6167.161
-- Chrome Driver 121.0.6167.85
-- Microsoft Edge 121.0.2277.112
-- Microsoft Edge Driver 121.0.2277.112
+- Google Chrome 121.0.6167.185
+- Chrome Driver 121.0.6167.184
+- Microsoft Edge 121.0.2277.106
```

Comment:
Version dropped down...

Reply:
@Alexey-Ayupov given that it is a minor I guess we can ignore it